### PR TITLE
Deploy prep

### PR DESCRIPTION
**Pull request by bencbernstein**

### CODE REVIEW

File: Apiary/MeViewController.swift [hunk 1]
```diff
@@ -53,24 +52,21 @@ final class MeViewController: UIViewController {
...
+    _ = CAGradientLayer().then {
+      $0.colors = [Palette.lightestBlue.color.cgColor, UIColor.white.cgColor]
+      $0.locations = [0.0, 0.50]
+      $0.frame = view.frame
+      view.layer.addSublayer($0)
     }
 
     userNameLabel = UILabel().then {
-      $0.text = User.fetch.firstName
+      let userName = (User.fetch.firstName != "") ? (User.fetch.firstName + "'s Progress") : "My Progress"
```
Comment (OLI9292) [inline Apiary/MeViewController.swift]: little better: `$0.text = User.fetch.firstName.isEmpty ? "My Progress" : "\(User.fetch.firstName)'s Progress"`

Comment (bencbernstein) [reply Apiary/MeViewController.swift]: `isEmpty` definitely references "" ?

Comment (OLI9292) [reply Apiary/MeViewController.swift]: `"".isEmpty` is `true`

**Comment by OLI9292** [comment]
otherwise looks G2G

**Pull request merged**